csvw/csvw.go

```diff
@@ -193,8 +192,8 @@ func Generate(ctx context.Context, metadata *dataset.Metadata, downloadURL, abou
 
 	// add dimension columns
 	if len(metadata.Dimensions) > 0 {
-		for i := 1; i < len(h); i++ {
-			l, err := newLabelColumn(i, apiDomain, h, metadata.Dimensions)
+		for i := range metadata.Dimensions {
```

Comment:
Any reason you couldn't use the full range loop here?
```
for _, d := range metadata.Dimensions{
  ...newLabelColumn(.., d)
}

Reply:
Nope, updated with your suggestion